recipe/run_conda_forge_build_setup_win

```diff
@@ -1,3 +1,8 @@
+
+:: 2 cores available on Appveyor workers: https://www.appveyor.com/docs/build-configuration/#build-environment
```

Comment:
This link appears to be gone. Here's the new one.

ref: https://www.appveyor.com/docs/build-environment/#build-vm-configurations